src/components/Checkout/Form.tsx

```diff
@@ -249,13 +288,15 @@ const Form: React.FC<{
 							<div className='mt-[42px] xl:mt-[3.889vh] flex flex-col sm:flex-row items-center gap-6 font-medium text-lg !leading-6'>
 								<button
 									type='submit'
-									className='max-sm:w-full bg-blue-primary text-primary py-[17px] px-[42px] rounded-full hover:brightness-105 transform transition duration-200 focus:ring-0 focus:outline-none'>
+									className='max-sm:w-full bg-blue-primary text-primary py-[17px] px-[42px] rounded-full hover:brightness-105 transform transition duration-200 focus:ring-0 focus:outline-none'
+								>
 									{ formSectionData.submitLabel }
 								</button>
 								<button
 									type='button'
-									onClick={ () => router.back() }
-									className='max-sm:w-full text-grey-primary border border-grey-primary transition duration-200 hover:brightness-105 py-[17px] px-[42px] rounded-full focus:ring-0 focus:outline-none'>
+									onClick={ () => router.replace('/') }
+									className='max-sm:w-full text-grey-primary border border-grey-primary transition duration-200 hover:brightness-105 py-[17px] px-[42px] rounded-full focus:ring-0 focus:outline-none'
```

Comment:
i think we already have a button component

Reply:
Code updated